### PR TITLE
Removed census household section titles

### PR DESCRIPTION
**Pull request by MIslam92**
## **Context of this PR**

Removing section headings from census household
Except for summary pages and printing answers after submission

## **How to review**

Run SR and look though Census Household to check that all section headings are removed and run tests

### CODE REVIEW

**Comment by jonnyshaw89** [comment]
@MIslam92 Did you look at just removing it from the theme?

**Comment by MIslam92** [comment]
@jonnyshaw89 I did however if I removed it from the census theme it would remove it from household, communal & individual? While the card only requests it to be removed from household?

File: data/en/census_household.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
             "blocks": [{
                     "type": "Questionnaire",
                     "id": "permanent-or-family-home",
-                    "title": "Who lives here?",
+                    "title": "",
```
Comment (shahi645) [inline data/en/census_household.json]: Get rid of these titles, to match rest of schema changes?

**Review by shahi645 — CHANGES_REQUESTED** [review]
Section headings still remain for individual sections but not on all questions. So need to remove of keep all. 
Also ` def complete_individual_section_person_1` failing in `test_census_household_submission_data`

File: tests/functional/spec/census-household.spec.js [hunk 2]
```diff
@@ -157,7 +157,7 @@ describe('Census Household', function () {
 
   });
 
-  it('Given a census household survey, When one person in household, Then persons name should appear on member details pages.', function () {
+ /* it('Given a census household survey, When one person in household, Then persons name should appear on member details pages.', function () {
```
Comment (jonnyshaw89) [inline tests/functional/spec/census-household.spec.js]: If we remove this then this needs to be deleted now commented

File: data/schema/schema_v1.json [hunk 3]
```diff
@@ -646,7 +646,6 @@
                     "additionalProperties": false,
                     "required": [
                       "id",
-                      "title",
```
Comment (jonnyshaw89) [inline data/schema/schema_v1.json]: You also need to add validation to say that if it is specified then it can't be empty.

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by shahi645 — APPROVED** [review]
LGTM, all section titles gone and tests pass 👍🏿

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
@MIslam92 Squashy Squishy

**Pull request merged**